src/CMakeLists.txt

```diff
@@ -722,7 +727,7 @@ ADD_CUSTOM_TARGET(tr
 	COMMAND ${Qt5_LUPDATE_EXECUTABLE} -noobsolete ${Canorus_UIC_Srcs} ${Canorus_Srcs} -ts ${Canorus_Translation_Srcs} lang/template.ts
 )
 set_source_files_properties(${Canorus_Translation_Srcs} PROPERTIES OUTPUT_LOCATION "lang")
-qt5_add_translation(Canorus_Translation_Bins ${Canorus_Translation_Srcs})
+#qt5_add_translation(Canorus_Translation_Bins ${Canorus_Translation_Srcs})
```

Comment:
By commenting out this, .qm files are never generated, because `${Canorus_Translation_Bins}` is now empty?

Reply:
You need to create src/lang in the build folder, then it will work.